app/src/main/java/org/cirdles/topsoil/app/TopsoilMainWindow.java

```diff
@@ -373,4 +376,27 @@ void emptyTable() {
         getCurrentTable().ifPresent(TsvTable::clear);
     }
 
+    @FXML
+    void openDocumentationLink() {
+        try {
+            if (Desktop.isDesktopSupported()) {
+                Desktop.getDesktop().browse(new URI("https://github.com/CIRDLES/Topsoil/wiki"));
+            }
+        } catch (IOException | URISyntaxException e) {
+            LOGGER.error(null, e);
```

Comment:
While it's okay to log the `URISyntaxException`, the `IOException` should be logged *and* notify the user.